packages/x-data-grid-pro/src/components/headerFiltering/GridHeaderFilterCell.tsx

```diff
@@ -113,7 +117,7 @@ const GridHeaderFilterCell = forwardRef<HTMLDivElement, GridHeaderFilterCellProp
     item,
     headerFilterMenuRef,
     InputComponentProps,
-    showClearIcon = true,
+    showClearIcon = false,
```

Comment:
_📝 Breaking change_

Reply:
Should only one of these two views (clear icon and menu item) be active at a time? So for example, hide the menu item when `showClearIcon` is `true`.